storage/Filesystems/SwapImpl.cc

```diff
@@ -105,30 +107,16 @@ namespace storage
 
     void
     Swap::Impl::do_mount(CommitData& commit_data, const CommitOptions& commit_options,
```

Comment:
Is this re-implementation needed? It looks like the same for `Mountable` base class.

Reply:
Indeed, this is not needed anymore since the real code move to another function.